source/development/feature-flags.html.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Feature flags in GOV.UK Frontend
+weight: 1
+last_reviewed_on: 2023-11-28
+review_in: 6 months
+---
+# Using feature flags in GOV.UK Frontend
+
+Feature flags are a way for us to release large scale or experimental work early by allowing users to opt in our out of a new, breaking change or feature. Use this guidance to decide if a feature flag is useful for your use case.
```

Comment:
```suggestion
Feature flags are a way for us to do a 'soft' release of large scale or experimental work by allowing users to opt in our out of a new, breaking change or feature. Read this guidance to decide whether a feature flag is a useful addition to your project.
```

Reply:
Experimenting with a way to remove 'early' which felt a bit orphaned within the sentence and some repetition of use/useful in the second sentence. Not essential, so it's a decision as to whether you think it flows better or not.

Reply:
I like it